code/modules/surgery/helpers.dm

```diff
@@ -145,10 +145,10 @@
 			if(covered_locations & HEAD)
 				return 0
 		if(BODY_ZONE_PRECISE_EYES)
-			if(covered_locations & HEAD || face_covered & HIDEEYES || eyesmouth_covered & GLASSESCOVERSEYES)
+			if(face_covered & HIDEFACE || face_covered & HIDEEYES || eyesmouth_covered & GLASSESCOVERSEYES || eyesmouth_covered & HEADCOVERSEYES)
```

Comment:
Should probably replace HIDEFACE check with a MASKCOVERSEYES check since not all masks with HIDEFACE actually cover your eyes (sterile mask, fake moustache, etc)

Reply:
HIDEFACE hides identity, meaning meaning it probably hides the eyes too, if the mask doesn't hide the eye, it probably shouldn't HIDEFACE.
meaning it's an issue with the masks, not this code